src/sentry_github/plugin.py

```diff
@@ -65,38 +84,58 @@ class GitHubPlugin(IssuePlugin):
     conf_key = 'github'
     project_conf_form = GitHubOptionsForm
     auth_provider = 'github'
+    create_issue_template = 'sentry_github/create_github_issue.html'
+    can_unlink_issues = True
+    can_link_existing_issues = True
 
     def is_configured(self, request, project, **kwargs):
         return bool(self.get_option('repo', project))
 
     def get_new_issue_title(self, **kwargs):
-        return 'Create GitHub Issue'
+        return 'Link GitHub Issue'
+
+    def get_unlink_issue_title(self, **kwargs):
+        return 'Unlink GitHub Issue'
 
     def get_new_issue_read_only_fields(self, **kwargs):
         group = kwargs.get('group')
         if group:
             return [{'label': 'Github Repository', 'value': self.get_option('repo', group.project)}]
         return []
 
+    def handle_api_error(self, request, error):
+        msg = _('Error communicating with GitHub: %s') % error
+        messages.add_message(request, messages.ERROR, msg)
+
     def get_allowed_assignees(self, request, group):
         try:
-            req = self.make_api_request(request, group, 'assignees')
+            url = self.build_api_url(group, 'assignees')
+            req = self.make_api_request(request, url)
             body = safe_urlread(req)
-        except requests.RequestException:
+        except requests.RequestException as e:
+            msg = unicode(e)
+            self.handle_api_error(request, msg)
             return tuple()
 
         try:
             json_resp = json.loads(body)
-        except ValueError:
+        except ValueError as e:
+            msg = unicode(e)
+            self.handle_api_error(request, msg)
             return tuple()
 
         if req.status_code > 399:
+            self.handle_api_error(request, json_resp.get('message', ''))
             return tuple()
 
         users = tuple((u['login'], u['login']) for u in json_resp)
 
         return (('', 'Unassigned'),) + users
 
+    def get_link_existing_issue_form(self, request, group, event, **kwargs):
```

Comment:
sure i can do that. i hadn't because originally i was doing something more github specific in that method, but i can just change [this](https://github.com/getsentry/sentry/pull/3495/files#diff-0e1bf00279c4b111e1f181be05cf3474R123) to be your suggestion.